tests/java-db-test/src/main/webapp/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script src="script.js"></script>
+    <title>DB Test</title>
+  </head>
+  <body onload="db()">
+  </body>
+</html>
```

Comment:
nit: Add newline at end of file.

Reply:
weird, to me it does have the newline, i'll try deleteing it and adding it